change-notes/1.19/analysis-javascript.md

```diff
@@ -22,5 +22,6 @@
 | Regular expression injection | Fewer false-positive results | This rule now identifies calls to `String.prototype.search` with more precision. |
 | Unbound event handler receiver | Fewer false-positive results | This rule now recognizes additional ways class methods can be bound. |
 | Remote property injection | Fewer results | The precision of this rule has been revised to "medium". Results are no longer shown on LGTM by default. |
+| Missing CSRF middleware | Fewer false-positive results | This rule now recognizes CSRF middleware from the [lusca](https://www.npmjs.com/package/lusca) package. |
```

Comment:
Hide the concrete library name:

> This rule now recognizes additional CSRF middlewares. 

And consider adding "protection"/"prevention":
> This rule now recognizes additional CSRF **protection** middlewares. 

(In https://github.com/Semmle/ql/commit/5781b518bcdd65ba6e788b225f795f001990c25f#diff-f48c5fc502455340119a9f5dfa158f06R8, we started to be less explicit about the exact libraries models)